jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/CloudFormation.kt

```diff
@@ -37,6 +39,13 @@ fun CloudFormationClient.executeChangeSetAndWait(stackName: String, changeSet: S
     }
 }
 
+fun CloudFormationClient.describeStack(stackName: String, callback: (Stack?) -> Unit) {
+    ApplicationManager.getApplication().executeOnPooledThread {
+        val stack = this.describeStacks { it.stackName(stackName) }.stacks()[0]
```

Comment:
You have the Stack marked as nullable, but you would throw an exception before that due to the [0] call. use firstOrNull

Reply:
done